src/ui/ResultLink/ResultLink.ts

```diff
@@ -359,15 +362,19 @@ export class ResultLink extends Component {
     if (!this.options.titleTemplate) {
       return this.result.title
         ? HighlightUtils.highlightString(this.result.title, this.result.titleHighlights, null, 'coveo-highlight')
```

Comment:
Are the other case correctly handling XSS injection ? the `HighlightUtils.highlightString` and `StreamHighlightUtils.highlightStreamText` ?

Reply:
Yep, both are using `HighlightUtils.highlightString` which escapes the value. I will include tests though,